tests/suites/test_suite_ssl.function

```diff
@@ -908,6 +908,34 @@ int mbedtls_move_handshake_to_state( mbedtls_ssl_context *ssl,
         }                                       \
     } while( 0 )
 
+void set_ciphersuite( mbedtls_ssl_config *conf, const char *cipher,
+                      int* forced_ciphersuite )
+{
+    const mbedtls_ssl_ciphersuite_t *ciphersuite_info;
+    forced_ciphersuite[0] = mbedtls_ssl_get_ciphersuite_id( cipher );
```

Comment:
Does anything bad happen if `cipher` is invalid (or not available in your configuration)? I'm thinking about making the code a bit more resilient against developer-made typos in the ciphersuite name.

Reply:
Done.